src/app/ApplicationTemplate.Mobile/Android/Resources/values/Colors.xml

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<resources>
+	<item name="backgroundColor" type="color">#001E60</item>
+	<item name="primaryColor" type="color">#0D59CD</item>
+	<item name="white" type="color">#FFFFFFFF</item>
```

Comment:
Where are they used? I know the other two are used in the Styles.xml file but I'm not sure about those one.

Reply:
Found out that `primaryColor` is used in `ic_launcher.xml`.

Reply:
Also found `backgroundColor` but not `white` so I will remove it.